src/PageBanner/PageBanner.scss

```diff
@@ -0,0 +1,47 @@
+.pgn__pageBanner-component {
+  width: 100%;
+  min-height: 36px;
+  max-height: 56px;
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.pgn__pageBanner__dark {
+  background-color: $dark-500;
+  color: white;
+}
+
+.pgn__pageBanner__light {
+  background-color: $light-400;
+  color: black;
+}
+
+.pgn__pageBanner__accentA {
+  background-color: $accent-a;
+  color: black;
+}
+
+.pgn__pageBanner__warning {
+  background-color: $warning-100;
+  color: black;
+}
+
+.pgn__pageBanner-content {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  padding: 8px 12px;
+  text-align: center;
+  font-size: $x-small-font-size;
+
+  @media only screen and (min-width: 768px) {
+    font-size: $small-font-size;
+  }
```

Comment:
nit: Might consider using the Bootstrap's responsive mixins here in case the breakpoint widths ever change, it'll be captured here as well:

```
@include media-breakpoint-up(md) {
  font-size: $small-font-size;
}
```